include/gmsh.h

```diff
@@ -35,17 +40,24 @@ class GMSH : public Mesh<Tdim, Tvertices> {
   void store_element_vertices();
 
   //! Compute material points from element coordinate map
-  void compute_material_points();
+  void compute_material_points(const unsigned ngauss_points);
```

Comment:
`unsigned` pass by value, no const necessary.

Reply:
why not pass by value for an `unsigned`?